cmake/PostgreSQLExtension.cmake

```diff
@@ -318,16 +318,30 @@ if [ -d \"$_dir/${NAME}\" ]; then
   _dir=\"$_dir/${NAME}\"
 fi
 # Create file (using $$ for pid to avoid race conditions)
+
 for f in $(ls \"$_dir/\"*.sql | sort -V)
   do
+  filename=$(basename \"$f\")
+  if [[ $filename =~ ([0-9]+)_ ]]; then
+    migration_number=\${BASH_REMATCH[1]}
+    if grep -q \" $migration_number \" <<< \"$unique_versions\"; then
+      echo \"Error: Duplicate migration number found: $migration_number in filename: $filename\" >&2
+      exit 1
+    else
+      unique_versions+=\" $migration_number \"
+    fi
+  else
+    echo \"Error: Unable to extract migration version from filename: $filename\" >&2
```

Comment:
I just realised that we have some migration files that don't follow this naming scheme, like `libpgaug_test.sql`.

Is that intentioned? Should I go ahead and rename those?

Reply:
Unintentional, renaming is good!